opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableCreation.jsx

```diff
@@ -202,13 +222,15 @@ const stixCyberObservableMutation = graphql`
     }
   }
 `;
-
-const stixCyberObservableValidation = () => Yup.object().shape({
-  x_opencti_score: Yup.number().nullable(),
-  x_opencti_description: Yup.string().nullable(),
-  createIndicator: Yup.boolean(),
-});
-
+let validObservables = 0;
+let errorObservables = 0;
+let totalObservables = 0;
+const error_array = [];
```

Comment:
So i moved these to be inside the const StixCyberObservableCreation block, but we need to use them as lets. If we convert them to use.States() then they are not updated because of the async and promises.

Reply:
ack